packages/model-viewer/src/three-components/CachingGLTFLoader.ts

```diff
@@ -161,6 +162,7 @@ export class CachingGLTFLoader {
 
     const gltf = await cache.get(url)!;
 
+    const duplicate: Mesh[] = [];
```

Comment:
The name here is a little ambiguous to me. Would you consider something a bit more descriptive like `meshesToDuplicate`? 🙏